tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/AspNetCore5AsmAttackerBlocking.cs

```diff
@@ -0,0 +1,139 @@
+// <copyright file="AspNetCore5AsmAttackerBlocking.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_0_OR_GREATER
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Datadog.Trace.AppSec;
+using Datadog.Trace.AppSec.Rcm.Models.AsmFeatures;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+using Action = Datadog.Trace.AppSec.Rcm.Models.Asm.Action;
+
+namespace Datadog.Trace.Security.IntegrationTests.Rcm;
+
+public class AspNetCore5AsmAttackerBlocking : RcmBase
+{
+    private const string AsmProduct = "ASM";
+
+    public AspNetCore5AsmAttackerBlocking(AspNetCoreTestFixture fixture, ITestOutputHelper outputHelper)
+        : base(fixture, outputHelper, enableSecurity: true, testName: nameof(AspNetCore5AsmAttackerBlocking))
+    {
+        SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "1");
+        SetEnvironmentVariable("DD_APPSEC_WAF_DEBUG", "0");
+    }
+
+    [Fact]
+    [Trait("RunOnWindows", "True")]
+    public async Task TestSuspiciousAttackerBlocking()
+    {
+        List<KeyValuePair<string, string>> headersAttacker = new()
+        {
+            new KeyValuePair<string, string>("http.client_ip", "34.65.27.85"),
+            new KeyValuePair<string, string>("X-Real-Ip", "34.65.27.85"),
+            new KeyValuePair<string, string>("accept-encoding", "identity"),
+            new KeyValuePair<string, string>("x-forwarded-for", null),
+        };
+
+        List<KeyValuePair<string, string>> headersRegular = new()
+        {
+            new KeyValuePair<string, string>("X-Real-Ip", null),
+            new KeyValuePair<string, string>("accept-encoding", "identity"),
+            new KeyValuePair<string, string>("x-forwarded-for", null),
+        };
+
+        var headersAttackerArachni = new List<KeyValuePair<string, string>>(headersAttacker)
+        {
+            new KeyValuePair<string, string>("User-Agent", "Arachni/v1"),
+        };
+
+        var headersRegularArachni = new List<KeyValuePair<string, string>>(headersRegular)
+        {
+            new KeyValuePair<string, string>("User-Agent", "Arachni/v1"),
+        };
+
+        var headersAttackerScanner = new List<KeyValuePair<string, string>>(headersAttacker)
+        {
+            new KeyValuePair<string, string>("User-Agent", "dd-test-scanner-log-block"),
+        };
+
+        var headersRegularScanner = new List<KeyValuePair<string, string>>(headersRegular)
+        {
+            new KeyValuePair<string, string>("User-Agent", "dd-test-scanner-log-block"),
+        };
+
+        SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "1");
```

Comment:
is this meant to stay?

Reply:
No, that was only for testing. I have deleted it.